src/tool/ToolArgs.cpp

```diff
@@ -84,6 +84,20 @@ R"(
     "inputs",
     llvm::cl::Sink,
     llvm::cl::desc("The path to the compilation database."))
+
+, generateCompilationDatabase(
+    "generate-compilation-database",
+    llvm::cl::desc("Generate the compilation database."),
+    llvm::cl::init(false))
+
+, cmakePath(
+    "cmake",
+    llvm::cl::desc(R"(The path to the cmake executable.)"))
+
+, cmakeListsPath(
```

Comment:
We just need the input path. There's no use for inputPath and cmakeListsPath at the same time because they're mutually exclusive.